.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: java
 
+jdk:
+  - oraclejdk8
```

Comment:
Can we still make sure the rest works with 6/7? I want to avoid an accidental regression here, as 2/3 of our users still are on 6.